lib/vault/api/logical.rb

```diff
@@ -40,7 +40,7 @@ def list(path)
     # @param [String] path
     #   the path to read
     #
-    # @return [Secret, nil]
+    # @return [Vault::Secret, nil]
```

Comment:
Np.  I didn't see a documentation tool in the gemspec but was trying to align the expectations with the other documentation which fully qualify the Secret object.